tests/functional/strategy/test_attack_vector.py

```diff
@@ -0,0 +1,54 @@
+from brownie import Wei, reverts
```

Comment:
Rename this file to ` tests/functional/strategy/test_attacks.py`

Reply:
fixed